lib/stimulus_reflex/sanity_checker.rb

```diff
@@ -53,14 +62,11 @@ def gem_version
   end
 
   def javascript_package_version
-    return @_js_version if defined?(@_js_version)
-    @_js_version = find_javascript_package_version
+    @_js_version ||= find_javascript_package_version
```

Comment:
Do we actually need the memoization? We could save a method if we don't memoize :-)

Reply:
It's actually used more than the gem version equivalent.